src/net_processing.cpp

```diff
@@ -2267,6 +2355,29 @@ static void ProcessGetCFCheckPt(CNode& peer, CDataStream& vRecv, const CChainPar
     connman.PushMessage(&peer, std::move(msg));
 }
 
+/**
+ * Announce transactions a peer is missing after reconciliation is done.
+ * No need to add transactions to peer's filter or do checks
+ * because it was already done when adding to the reconciliation set.
+ */
+void static AnnounceTxs(const std::vector<uint256>& remote_missing_wtxids, CNode& pto, const CNetMsgMaker& msgMaker, CConnman& connman)
+{
+    if (remote_missing_wtxids.size() != 0) {
```

Comment:
Consider inverting this if condition to make this a guard clause and avoid deep indentations below.

```suggestion
    if (remote_missing_wtxids.size() == 0) return;
```